igtdetect/pdf2gloss.py

```diff
@@ -122,8 +133,20 @@ def detect_igts(input_path, temp_path, model_path, config_path, base_path):
         logging.error('igt-detect failed')
     return analyzed_features_path
 
+def match_dois(IGT_list, dois):
+    '''
+    Matches DOIs to IGT objects using the source filename (without the extension)
+    returns an IGT_list with the DOIs supplemented
+    '''
+    for IGT in IGT_list:
```

Comment:
purely style: avoid all-caps for variable names. `igt` may look odd but `IGT` would seem like a constant, because it's such a strong convention.